packages/component-base/src/tooltip-controller.d.ts

```diff
@@ -5,6 +5,20 @@
  */
 import { SlotController } from './slot-controller.js';
 
+type TooltipPosition =
```

Comment:
This  copy-paste from `tooltip` package is used to keep controller separate from the component.